changelog/issue-1111.md

```diff
@@ -0,0 +1,3 @@
+level: patch
+reference: issue 1875
+---
```

Comment:
Add a message similar to the changelog message in https://github.com/taskcluster/taskcluster/pull/1893/files.